R-package/tests/testthat/test_basic.R

```diff
@@ -1534,6 +1534,60 @@ test_that("lgb.train() works with integer, double, and numeric data", {
   }
 })
 
+test_that("lgb.train() updates params based on keyword arguments", {
+  dtrain <- lgb.Dataset(
+    data = matrix(rnorm(400L), ncol =  4L)
+    , label = rnorm(100L)
+  )
+
+  # defaults from keyword arguments should be used if not specified in params
+  invisible(
```

Comment:
these uses of `invisible(capture.output(...))` suppress log messages from being printed, per #4862